pom.xml

```diff
@@ -45,8 +56,8 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>2.3.2</version>
                 <configuration>
-                    <source>1.6</source>
-                    <target>1.6</target>
+                    <source>1.8</source>
```

Comment:
Oh okay that makes sense!  I will make the adjustments to restore compatibility with 1.7.  The change is very small if I remember correctly   Thanks!